data/modules/TradeShips/Events.lua

```diff
@@ -159,26 +142,31 @@ Event.Register("onShipDocked", onShipDocked)
 
 local onShipUndocked = function (ship, starport)
 	if Core.ships[ship] == nil then return end
+	local trader = Core.ships[ship]
+	
+	if trader.starport.type == "STARPORT_ORBITAL" then
+		-- fly to the limit of the starport frame
+		ship:AIFlyTo(starport)
+	else
+        -- enter low orbit of first system's star
+		ship:AIEnterLowOrbit(trader.starport.path:GetStarSystem():GetStars()[1].body)
+		Trader.assignTask(ship, Game.time + 10, 'hyperjumpAtDistance')
+	end
 
-	-- fly to the limit of the starport frame
-	ship:AIFlyTo(starport)
-
-	Core.ships[ship]['status'] = 'outbound'
+	trader['status'] = 'outbound'
 end
 Event.Register("onShipUndocked", onShipUndocked)
 
 local onAICompleted = function (ship, ai_error)
 	if Core.ships[ship] == nil then return end
 	local trader = Core.ships[ship]
 	if ai_error ~= 'NONE' then
-		Core.log:add(ship, 'AICompleted: Error: '..ai_error..' Status: '..trader.status) end
-
+		Core.log:add(ship, 'AICompleted: Error: '..ai_error..' Status: '..trader.status) 
```

Comment:
There is an extra space at the end of the line.

Reply:
Not anymore.